flutter/cpp/flutter/BUILD

```diff
@@ -41,7 +41,9 @@ cc_library(
     hdrs = ["ios_linker_workaround.h"],
     deps = [
         "//flutter/cpp/flutter:bridge",
-        "//mobile_back_tflite/cpp/backend_tflite:tflite_c",
+        # TODO(anhappdev) Add option to choose between TFLite and CoreML
```

Comment:
I am trying to integrate a mockup backend to the app first and will implement the functions in later steps but currently get stuck on how to make the iOS app can work with both TFLite and Core ML backend.

Reply:
Currently on iOS we link tflite backend statically and call its exported functions as functions of the main application.
It was probably was easier to do when the iOS version of the app was initially implemented.
To make it possible to support several backends we should link backends dynamically.
I'll look into this. https://github.com/mlcommons/mobile_app_open/issues/new